go/pkg/log/server/property_test.go

```diff
@@ -168,19 +168,32 @@ func compareModelLogRecordToLogRecord(t *rapid.T, modelLogRecord ModelLogRecord,
 // is the same in both the model and the SUT
 func (suite *LogServerTestSuite) invariantLogsAreTheSame(ctx context.Context, t *rapid.T) {
 	for id, model_log := range suite.model.CollectionData {
-		pulled_log, err := suite.lr.PullRecords(ctx, id.String(), 0, len(model_log), time.Now().UnixNano())
+		next_offset := suite.model.CollectionCompactionOffset[id] + 1
```

Comment:
Understanding: Why is this +1 ?

Reply:
because the compaction offset is the offset of the last compacted record. So +1 is the offset of the first record that needs to be pulled